README.md

```diff
@@ -1,3 +1,94 @@
 # promises-debugging
 
-Placeholder
+Promises have been available to Node.js developers for several years in various
+forms (as a built-in feature of the language or as npm modules). However,
+debugging errors created from promises tasks can still present some challenges
+in some use cases.
+
+As a result, an _informal_ working group was formed with the goal to make the
```

Comment:
working groups are by definition formal as far as I know - "team" works I believe.

Reply:
agreed, team is the right wording to use.